src/Containers/Chart.tsx

```diff
@@ -137,7 +137,7 @@ color:${color};
 const HeaderLeftSubtitle = styled.div`
     ${({ theme }): string => `
 font-size:${theme.font.size.small};
-color:${theme.colors.status.complete};
+color:${theme.colors.statusColors.green};
```

Comment:
@Ampatte2 is this correct? If so all good

Reply:
yes, the interface (of the theme) changed so I had to update my code to make it to work again

Reply:
This is correct, statusColors is generic colors to avoid odd naming when used.